environment.yml

```
@@ -0,0 +1,12 @@
+name: autometa-nf
+
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+
+dependencies:
+  - nextflow>=20.10
+  - bioconda::nf-core>=2.1
```

need to pin nf-core to the specific version, otherwise it'll complain about schema differences
```suggestion
  - bioconda::nf-core==2.1
```